frontend/sw360-portlet/src/main/webapp/html/utils/includes/linkedReleasesEdit.jspf

```diff
@@ -19,7 +19,7 @@
            <col style="width: 20%;" />
            <col style="width: 20%;" />
            <col style="width: 20%;" />
-           <col style="width: 45px;" /> <!-- width of one action (25) with padding of 10 for the column-->
+           <col style="width: 100px;" /> <!-- width of one action (25) with padding of 10 for the column-->
```

Comment:
Is the comment still valid?

Reply:
(from discussion) 70? 95? 120? maybe have a look at other tables

Reply:
Thanks for the suggestion.width: 80px looks feasible and any value less then that is distorting the alignment.So I will be using <col style="width: 80px;" />